ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/142__dsc_components.robot

```diff
@@ -184,6 +180,18 @@ Suite Teardown
     [Documentation]    Suite Teardown
     RHOSi Teardown
 
+Check Controller Conditions Are Accomplished
+    [Documentation]    Wait for the conditions related to a specific controller are accomplished
+    [Arguments]    ${controller}
+
+    @{d_obj}=  OpenShiftLibrary.Oc Get  kind=Deployment  name=${controller}    namespace=${APPLICATIONS_NAMESPACE}
+    &{d_obj_dictionary}=  Set Variable  ${d_obj}[0]
+    ${cpu_limit}=    Set Variable    ${d_obj_dictionary.spec.template.spec.containers[0].resources.limits.cpu}
+    ${memory_limit}=    Set Variable    ${d_obj_dictionary.spec.template.spec.containers[0].resources.limits.memory}
+    Should Match    ${d_obj_dictionary.spec.template.spec.containers[0].resources.limits.cpu}    ${cpu_limit}
```

Comment:
No Berto, there is a default value overwritten when we patch. Then, that variable has to match because otherwise, that means that the Operator restored it to its default value and it shouldn't

Reply:
then the values must be captured in different moments: before and after the patch. While here the 2 values are always the same because taken at the same moment: trying to match it and itself

Reply:
I think @bdattoma is right. We should change lines 191 and 192:

    Should Match    ${d_obj_dictionary.spec.template.spec.containers[0].resources.limits.cpu}    600m
    Should Match    ${d_obj_dictionary.spec.template.spec.containers[0].resources.limits.memory}    600Gi